test/libsolidity/syntaxTests/operators/custom/duplicated_operator_binding.sol

```diff
@@ -0,0 +1,13 @@
+type Int is int32;
+
+using {add as +, add as +} for Int;
```

Comment:
Shouldn't this create an error? What is the semantic right now?

Reply:
Not sure whether this was discussed somewhere else, but at least right now I would prefer that this is forbidden. Binding the same function multiple times to the same operator should be forbidden.

@cameel @chriseth

Reply:
Or is it basically just having the same semantic as a single operator binding? maybe that's then ok. But at least it will create some confusion, if someone sees such construct the first time - because the semantic is probably not that clear.

Reply:
Not sure. I'd be inclined to make this an error if possible. But it might be tricky. We can't just completely disallow it because there are cases where we do need it - e.g. binary and unary minus.